pkg/volume/glusterfs/glusterfs.go

```diff
@@ -819,6 +840,7 @@ func (p *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 		"kubernetes.io/createdby":           heketiAnn,
 		glusterTypeAnn:                      "file",
 		"Description":                       glusterDescAnn,
+		v1.MountOptionAnnotation:            "auto_unmount",
```

Comment:
This option is introduced as a fix in gluster fuse bridge, so we would like to enable it for every upcoming version, so the annotation set in the pv and filtering out clients < 3.11 to make sure gluster mount works for existing clients till they upgrade.